[2025-04-02T08:05:35.748+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_b.load_data scheduled__2025-04-01T08:00:00+00:00 [queued]>
[2025-04-02T08:05:35.752+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_b.load_data scheduled__2025-04-01T08:00:00+00:00 [queued]>
[2025-04-02T08:05:35.753+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-04-02T08:05:35.758+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): load_data> on 2025-04-01 08:00:00+00:00
[2025-04-02T08:05:35.765+0000] {standard_task_runner.py:60} INFO - Started process 1048 to run task
[2025-04-02T08:05:35.767+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'dag_b', 'load_data', 'scheduled__2025-04-01T08:00:00+00:00', '--job-id', '55', '--raw', '--subdir', 'DAGS_FOLDER/dynamic_dag.py', '--cfg-path', '/tmp/tmpxo1xunor']
[2025-04-02T08:05:35.768+0000] {standard_task_runner.py:88} INFO - Job 55: Subtask load_data
[2025-04-02T08:05:35.790+0000] {task_command.py:423} INFO - Running <TaskInstance: dag_b.load_data scheduled__2025-04-01T08:00:00+00:00 [running]> on host f4b0101fc635
[2025-04-02T08:05:35.820+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_b' AIRFLOW_CTX_TASK_ID='load_data' AIRFLOW_CTX_EXECUTION_DATE='2025-04-01T08:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-01T08:00:00+00:00'
[2025-04-02T08:05:35.825+0000] {base.py:83} INFO - Using connection ID 'postgres_default' for task execution.
[2025-04-02T08:05:35.909+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-04-02T08:05:35.914+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=dag_b, task_id=load_data, execution_date=20250401T080000, start_date=20250402T080535, end_date=20250402T080535
[2025-04-02T08:05:35.950+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-04-02T08:05:35.959+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-02T10:26:00.937+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_b.load_data scheduled__2025-04-01T08:00:00+00:00 [queued]>
[2025-04-02T10:26:00.941+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_b.load_data scheduled__2025-04-01T08:00:00+00:00 [queued]>
[2025-04-02T10:26:00.942+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-04-02T10:26:00.947+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): load_data> on 2025-04-01 08:00:00+00:00
[2025-04-02T10:26:00.951+0000] {standard_task_runner.py:60} INFO - Started process 149 to run task
[2025-04-02T10:26:00.953+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'dag_b', 'load_data', 'scheduled__2025-04-01T08:00:00+00:00', '--job-id', '77', '--raw', '--subdir', 'DAGS_FOLDER/dynamic_dag.py', '--cfg-path', '/tmp/tmpglk1zgaq']
[2025-04-02T10:26:00.954+0000] {standard_task_runner.py:88} INFO - Job 77: Subtask load_data
[2025-04-02T10:26:00.978+0000] {task_command.py:423} INFO - Running <TaskInstance: dag_b.load_data scheduled__2025-04-01T08:00:00+00:00 [running]> on host 247d35c01501
[2025-04-02T10:26:01.024+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_b' AIRFLOW_CTX_TASK_ID='load_data' AIRFLOW_CTX_EXECUTION_DATE='2025-04-01T08:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-01T08:00:00+00:00'
[2025-04-02T10:26:01.031+0000] {base.py:83} INFO - Using connection ID 'postgres_default' for task execution.
[2025-04-02T10:26:01.176+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-04-02T10:26:01.189+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=dag_b, task_id=load_data, execution_date=20250401T080000, start_date=20250402T102600, end_date=20250402T102601
[2025-04-02T10:26:01.210+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-04-02T10:26:01.225+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
